packages/vscode-extension/messages.json

```diff
@@ -54,6 +54,8 @@
 	"ERRORS_NO_ACTIVE_FILE_OR_WORKSPACE": "There is no active file or the active file is not in a workspace.",
 	"ERRORS_SDK_JAVA_VERSION_NOT_COMPATIBLE": "Your Java version {0} is not compatible with SuiteCloud Extension for Visual Studio Code. Install Oracle JDK version {1}.",
 	"ERRORS_SDK_JAVA_VERSION_NOT_INSTALLED": "Java is not installed on this machine. Install Oracle JDK version {0} to run SuiteCloud Extension for Visual Studio Code.",
+	"ERRORS_SDF_OBJECT_MUST_BE_IN_OBJECTS_FOLDER": "The SDF object file should be in the Objects folder.",
+	"ERRORS_UPDATE_FILE_FOLDER_RESTRICTION": "The file should be in a supported folder inside the FileCabinet folder. For information about the supported folders, see https://system.netsuite.com/app/help/helpcenter.nl?fid=section_4724963992.html.",
```

Comment:
The key should be `ERRORS_UPLOAD_FILE_FOLDER_RESTRICTION` but even I think that this 2 new errors should be placed in their commands keys `UPLOAD_FILE` and `UPDATE_OBJECT` because they are not generic errors as the other ones.
And as the other message I think it should have some kind of reference to the upload file command.

Reply:
Updated.